BAZProject/MVVM/Controllers/HomeController/HomeView.swift

```diff
@@ -0,0 +1,157 @@
+//
+//  HomeView.swift
+//  BAZProject
+//
+//  Created by victor manzanero on 07/11/22.
+//
+
+import UIKit
+import Bond
+
+class HomeViewController: UIViewController {
+
+    @IBOutlet weak var movieTable: UITableView!
+    @IBOutlet weak var loadingView: UIActivityIndicatorView!
+    @IBOutlet weak var searchBar: UISearchBar!
+    @IBOutlet weak var myPicker: UIPickerView!
+    @IBOutlet weak var myToolbar: UIToolbar!
+    @IBOutlet weak var btnDone: UIBarButtonItem!
+    @IBOutlet weak var btnCancel: UIBarButtonItem!
+    
+    var viewModel: HomeViewModel?
+    lazy var alert: CustomAlertViewController = {
+        return CustomAlertViewController()
+    }()
+    
+    override func viewDidLoad() {
+        self.config()
+        self.setupObservables()
+        self.setupObservablesAction()
```

Comment:
No es necesario que agregues `self.` antes de cada método.
Faltó llamar a `super.viewDidLoad()`
```suggestion
        super.viewDidLoad()
        config()
        setupObservables()
        setupObservablesAction()
```